### PR TITLE
Added Bluebert code and additional training parameters for BioBERT training

### PR DESCRIPTION
**Pull request by itsmemala**

### CODE REVIEW

File: src/contradictory_claims/data/make_dataset.py [hunk 1]
```diff
@@ -27,19 +31,53 @@ def load_multi_nli(train_path: str, test_path: str):
...
+        x_train = x_train.to_numpy()
+        x_test_1 = '[CLS]' + multinli_test_data.sentence1 + '[SEP]' + multinli_test_data.sentence2 +\
+                   '[SEP]' + 'CON'
+        x_test_2 = '[CLS]' + multinli_test_data.sentence1 + '[SEP]' + multinli_test_data.sentence2 +\
+                   '[SEP]' + 'ENT'
+        x_test_3 = '[CLS]' + multinli_test_data.sentence1 + '[SEP]' + multinli_test_data.sentence2 +\
+                   '[SEP]' + 'NEU'
+        x_test = x_test_1.append(x_test_2)
+        x_test = x_test.append(x_test_3)
+        x_test = x_test.to_numpy()
```
Comment (dnsosa) [inline src/contradictory_claims/data/make_dataset.py:62]: Feels like this functionality is copy-pasted and thus could be made more concise with helper functions. For the sake of merging now it's fine, but if we wanted to clean up and refactor in future, this is something to keep in mind

**Review by dnsosa — APPROVED** [review]

**Comment by dnsosa** [comment]
Do you know what's going on with Travis? Looks like tox is timing out after 20 minutes

File: src/contradictory_claims/data/make_dataset.py (hunk 1 above)
Comment (itsmemala) [reply src/contradictory_claims/data/make_dataset.py:62]: Good point! Agree.

**Comment by itsmemala** [comment]
Hmm weird...comparing against the travis build in my repo (which isn't timing out), I think what's happening is that it's timing out while running test_train_model.py...

**Comment by itsmemala** [comment]
And that's the one that was failing in my build...so I think maybe it's running successfully but just timing out because it's taking too long?

**Pull request merged**